test/steps/kogitoexplainability.go

```diff
@@ -0,0 +1,130 @@
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package steps
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/cucumber/godog"
+	"github.com/kiegroup/kogito-cloud-operator/test/framework"
+	"github.com/kiegroup/kogito-cloud-operator/test/steps/mappers"
+	bddtypes "github.com/kiegroup/kogito-cloud-operator/test/types"
+)
+
+/*
+	DataTable for Explainability:
+	| runtime-request | cpu/memory  | value                     |
+	| runtime-limit   | cpu/memory  | value                     |
+	| runtime-env     | varName     | varValue                  |
+	| kafka           | externalURI | kafka-bootstrap:9092      |
+	| kafka           | instance    | external-kafka            |
+*/
+
+func registerKogitoExplainabilityServiceSteps(ctx *godog.ScenarioContext, data *Data) {
+	ctx.Step(`^Install Kogito Explainability with (\d+) replicas$`, data.installKogitoExplainabilityServiceWithReplicas)
+	ctx.Step(`^Install Kogito Explainability with (\d+) replicas with configuration:$`, data.installKogitoExplainabilityServiceWithReplicasWithConfiguration)
+	ctx.Step(`^Kogito Explainability has (\d+) pods running within (\d+) minutes$`, data.kogitoExplainabilityHasPodsRunningWithinMinutes)
+	ctx.Step(`^Explainability result is available in the Trusty service within (\d+) minutes$`, data.explainabilityResultIsAvailable)
+}
+
+func (data *Data) installKogitoExplainabilityServiceWithReplicas(replicas int) error {
+	explainability := framework.GetKogitoExplainabilityResourceStub(data.Namespace, replicas)
+	return framework.InstallKogitoExplainabilityService(data.Namespace, framework.GetDefaultInstallerType(), &bddtypes.KogitoServiceHolder{KogitoService: explainability})
+}
+
+func (data *Data) installKogitoExplainabilityServiceWithReplicasWithConfiguration(replicas int, table *godog.Table) error {
+	explainability := &bddtypes.KogitoServiceHolder{
+		KogitoService: framework.GetKogitoExplainabilityResourceStub(data.Namespace, replicas),
+	}
+
+	if err := mappers.MapKogitoServiceTable(table, explainability); err != nil {
+		return err
+	}
+
+	return framework.InstallKogitoExplainabilityService(data.Namespace, framework.GetDefaultInstallerType(), explainability)
+}
+
+func (data *Data) kogitoExplainabilityHasPodsRunningWithinMinutes(podNb, timeoutInMin int) error {
+	return framework.WaitForKogitoExplainabilityService(data.Namespace, podNb, timeoutInMin)
+}
+
+type executionsResponse struct {
+	Executions []execution `json:"executions"`
```

Comment:
When I retrieved content of <trusty URL>/executions then the response is:
`{\"total\":1,\"limit\":100,\"offset\":0,\"headers\":[{\"executionId\":\"ac61b2ac-d02d-4349-b82d-72656c1ca8de\",\"executionDate\":\"2020-09-07T13:23:00+0000\",\"executionSucceeded\":true,\"executorName\":null,\"executedModelName\":\"LoanEligibility\",\"executedModelNamespace\":\"https://github.com/kiegroup/kogito-examples/dmn-quarkus-listener-example\",\"executionType\":\"DECISION\"}]}"}
`

So, should the JSON marshalling hint be adjusted?
```suggestion
	Executions []execution `json:"headers"`
```

Reply:
Yep, thank you!